packages/mui-material/src/Tabs/Tabs.test.js

```diff
@@ -608,12 +600,11 @@ describe('<Tabs />', () => {
       expect(container.querySelectorAll(`.${classes.scrollButtonsHideMobile}`)).to.have.lengthOf(0);
     });
 
-    it('should handle window resize event', function test() {
+    it('should handle window resize event', async function test() {
```

Comment:
We can also remove this test, since we do not update scroll buttons on window's `resize` event. It is handled by IntersectionObserver API on resizing the window.